lib/lndclient/LndClient.ts

```diff
@@ -72,6 +73,25 @@ class LndClient extends BaseClient {
     });
   }
 
+  private connect = (lndCLient: LndClient, subscribeInvoices: boolean = false) => {
+    const { host, port } = lndCLient.config;
+
+    lndCLient.lightning = new LightningClient(`${host}:${port}`, lndCLient.credentials);
+
+    lndCLient.getInfo().then((getInfoResponse) => {
+      if (getInfoResponse && getInfoResponse.numActiveChannels >= 1) {
+        lndCLient.setStatus(ClientStatus.CONNECTION_VERIFIED);
+        if (subscribeInvoices) {
+          lndCLient.subscribeInvoices();
+        }
+      }
+    }).catch((err: any) => {
+      lndCLient.logger.error(`could not fetch info from lnd host, error: ${err}, retrying in 5000 ms`);
+      lndCLient.setStatus(ClientStatus.DISABLED);
```

Comment:
I don't think we need to set the status to disabled here, because if we are attempting to connect that would mean that the status should already be set to DISABLED.